src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py

```diff
@@ -44,7 +45,16 @@ def __init__(
         self.verbose = verbose
         self.photon_series_type = photon_series_type
 
-    def get_metadata_schema(self) -> dict:
+    def get_metadata_schema(self, photon_series_type: Literal["OnePhotonSeries", "TwoPhotonSeries"] = None) -> dict:
```

Comment:
```suggestion
    def get_metadata_schema(self, photon_series_type: Optional[Literal["OnePhotonSeries", "TwoPhotonSeries"]] = None) -> dict:
```

I know it's just temporary; still including